services/src/unit-test/java/uk/gov/moj/sdt/services/TargetApplicationSubmissionServiceTest.java

```diff
@@ -1013,106 +951,114 @@ public void processCCDReferenceFailsOnUnSupportedRequest() {
-                                            EasyMock.anyBoolean());
-        EasyMock.expectLastCall();
+        this.mockMessageWriter.queueMessage(isA(ISdtMessage.class), isA(String.class), anyBoolean());
 
         final IGlobalParameter maxForwardingAttemptsParam = new GlobalParameter();
         maxForwardingAttemptsParam.setName("MAX_FORWARDING_ATTEMPTS");
         maxForwardingAttemptsParam.setValue("3");
-        expect(this.mockCacheable.getValue(IGlobalParameter.class, "MAX_FORWARDING_ATTEMPTS")).andReturn(
-            maxForwardingAttemptsParam);
-
-        EasyMock.replay(mockIndividualRequestDao);
-        EasyMock.replay(mockCmcConsumerGateway);
-        EasyMock.replay(xmlReader);
-        EasyMock.replay(mockMessageWriter);
-        EasyMock.replay(ccdReferenceValidator);
-        EasyMock.replay(mockCacheable);
 
-        // Setup dummy target response
         SdtContext.getContext().setRawInXml("response");
 
         this.targetAppSubmissionService.processRequestToSubmit(sdtRequestRef, false);
 
-        EasyMock.verify(mockIndividualRequestDao);
-        EasyMock.verify(mockMessageWriter);
-        EasyMock.verify(mockCmcConsumerGateway);
-        EasyMock.verify(mockCacheable);
-        EasyMock.verify(xmlReader);
-        EasyMock.verify(ccdReferenceValidator);
-
         Assert.assertEquals("Individual Request status not as expected",
                             IIndividualRequest.IndividualRequestStatus.FORWARDED.getStatus(),
                             individualRequest.getRequestStatus());
+
+        verify(mockCmcConsumerGateway).individualRequest(any(IIndividualRequest.class), anyLong(), anyLong());
+    }
+
+    @Test
+    public void processClaimRequestWhenBulkCustomerReadyForAlternateService() {
+        final String sdtRequestRef = "TEST_1";
+        final IIndividualRequest individualRequest = new IndividualRequest();
+        individualRequest.setSdtRequestReference(sdtRequestRef);
+        individualRequest.setRequestStatus("Received");
+        setUpIndividualRequest(individualRequest);
+        individualRequest.setRequestType(CLAIM.getType());
+        individualRequest.setRequestPayload("Test Xml");
+        individualRequest.setRequestStatus("Forwarded");
+        individualRequest.getBulkSubmission().getBulkCustomer().setReadyForAlternateService(true);
+
+        final IGlobalParameter individualReqProcessingDelay = new GlobalParameter();
+        individualReqProcessingDelay.setValue("10");
+        individualReqProcessingDelay.setName("MCOL_INDV_REQ_DELAY");
+
+        final IGlobalParameter connectionTimeOutParam = new GlobalParameter();
+        connectionTimeOutParam.setName("TARGET_APP_TIMEOUT");
+        connectionTimeOutParam.setValue("1000");
+
+        final IGlobalParameter receiveTimeOutParam = new GlobalParameter();
+        receiveTimeOutParam.setName("TARGET_APP_RESP_TIMEOUT");
+        receiveTimeOutParam.setValue("12000");
```

Comment:
Are these needed for the test?  They don't seem to be referenced after they've been declared.

Reply:
removed